base_export_manager/models/ir_exports_line.py

```diff
@@ -53,104 +62,106 @@ def _default_model1_id(self):
         """Default model depending on context."""
         return self.env.context.get("default_model1_id", False)
 
-    @api.multi
-    @api.depends("field1_id", "field2_id", "field3_id")
+    @api.one
```

Comment:
I prefer `@api.one` while it exists. And I repeat that it's still not clear that it will be deprecated.